### ISSUE TITLE
Update: Use CommandModule

### ISSUE THREAD

**Issue by sjrct** [halibot-extra/Meta#3]
We should use the command module for all modules that parse commands.

Relevant PR: [halibot/halibot#36](https://github.com/Halibot/halibot/pull/36)

**Comment by richteer** [comment]
Non-authoritative list ofcandidate modules for `CommandModule`. Not all are necessarily good fits for conversion, but it is worth taking a look at the following:
- [x] dice
- [ ] mastermind (probably not)
- [ ] quote
- [x] stackmemory
- [ ] tz
- [ ] emote
- [ ] timer (maybe?)
- [ ] btc
- [ ] reminders
- [ ] geoip
probably others...

Some tips:
 - look for modules that basically only ever react to a command
 - modules that already split command logic into functions should be trivial to convert
 - modules that have a lot of extra processing in `.receive()` may be a pain to convert
    - especially modules that also read bare strings or other sequences, unless you want to override `.receive()` carefully.
